docs/api.md

```diff
@@ -4680,3 +4682,226 @@ ScopeStatus defines the observed state of Scope.
         <td>true</td>
       </tr></tbody>
 </table>
+
+## TCOLogsPolicies
+<sup><sup>[↩ Parent](#coralogixcomv1alpha1 )</sup></sup>
+
+
+
+
+
+
+TCOLogsPolicies is the Schema for the tcologspolicies API.
```

Comment:
Please add a note in a prominent way, that this resource will override everything that is in the backend (i.e. if there are other policies, it will override them).